app/Http/Controllers/api/AdditionalTrainingListController.php

```diff
@@ -1,32 +1,37 @@
 <?php
 
 namespace App\Http\Controllers\api;
-
+use App\Models\Student;
+use App\Models\AdditionalTraining;
 use App\Http\Controllers\Controller;
 use Illuminate\Http\Request;
 
 class AdditionalTrainingListController extends Controller
 {
-    public function __invoke(){
-
-        $data = json_decode(file_get_contents(base_path('database/data/addtionalTraining.json')), true);
-        
-        $additionalTraining = $data['additionalTraining'];
-
-        $quantity = rand(2, min(7, count($additionalTraining))); 
-        $selectedProjects = [];
-    
-        while (count($selectedProjects) < $quantity) {
-            $randomProject = $additionalTraining[array_rand($additionalTraining)];
-            if (!in_array($randomProject, $selectedProjects)) {
-                $selectedProjects[] = $randomProject;
-            
-            }
-        
-        }
-    
-        return response()->json([
-            'additionalTraining' => $selectedProjects
-        ],200);
+    public function __invoke($uuid){
+
+        $student = Student::where('id', $uuid)->with('resume')->firstOrFail();
+
+        $resume = $student->resume;
+
+        $additionalTrainingIds = json_decode($resume->additional_trainings_ids);
+
+        $additionalTrainings = AdditionalTraining::findMany($additionalTrainingIds);
+
+        $additionalTrainingDetail = [
+            'additional_trainings' => $additionalTrainings->map(function ($additionalTraining) {
+                return [
+                    'uuid' => $additionalTraining->id,
+                    'course_name' => $additionalTraining->course_name,
+                    'study_center' => $additionalTraining->study_center,
+                    'course_beggining_year' => $additionalTraining->course_beggining_year,
+                    'course_ending_year' => $additionalTraining->course_ending_year,
+                    'duration_hrs' => $additionalTraining->duration_hrs,
+                ];
+            })
+        ];
+
+        return response()->json($additionalTrainingDetail);
     }
+
```

Comment:
![Array vacío](https://github.com/IT-Academy-BCN/ita-profiles-backend/assets/106141089/6335a541-61f5-4492-a1a9-292d4f4711b4)
**I already told you this but in swagger it returns an empty array...**

Reply:
Solved, the problem was mine for not doing migrate:fresh --seed

Reply:
Goog. Problem solved.